tests/unit/lms/services/lti_h_test.py

```diff
@@ -48,18 +87,6 @@ def test_it_upserts_the_GroupInfo_into_the_db(
             params=params,
         )
 
-    def test_it_doesnt_upsert_GroupInfo_into_the_db_if_upserting_the_group_fails(
-        self, group_info_service, h_api, lti_h_svc, h_group
-    ):
-        h_api.upsert_group.side_effect = HAPIError("Oops")
-
-        try:
-            lti_h_svc.sync([h_group])
-        except:  # pylint: disable=bare-except
-            pass
-
-        group_info_service.assert_not_called()
-
```

Comment:
Moved out and merged with `test_sync_catches_HAPIErrors` Not easy to spot in the diff.